src/grpc/keyprovider/message.rs

```diff
@@ -52,17 +52,20 @@ impl KeyProviderInput {
         Ok(())
     }
 
+    #[must_use]
```

Comment:
You might want to put this stuff in a separate PR. Should be trivial to merge.

Reply:
Done.